requirements.txt

```diff
@@ -0,0 +1,218 @@
+aiohttp==3.8.1
+aiosignal==1.2.0
+alabaster==0.7.12
+anki @ file:///home/rory/.cache/yay/anki/src/anki-2.1.49/bazel-bin/pylib/anki/anki-2.1.49-cp38-abi3-manylinux2014_x86_64.whl
```

Comment:
Hmm this doesn't look right... :laughing:

Reply:
I ran `pip freeze > requirements.txt` from the wrong virtual environment! Sorry for the confusion.

Reply:
ah gotcha! thanks, good to know what to look out for